pagopa/io-dev-api-server
# chore: [IOBP-804] Added feedback banner mock for the new payment success screen

## ⚠️ This PR depends on https://github.com/pagopa/io-services-metadata/pull/821

## Short description
This PR adds the feedback banner definition and its mock. This property is used to show the feedback banner when the user completes a payment successfully with the new payment flow.

## List of changes proposed in this pull request
- Updated content_specs definitions (N.B: When the dependent PR indicated above will be merged, the content_specs definitions have to be updated with the version deployed of io-services-metadata);
- Added `feedbackBanner` properties inside `newPaymentsSection`;

## How to test
With this PR runned on the dev-server, start the io app in DEV env. and complete a payment flow, you should be able to see a banner with the provided text inside the payment thank you page
